common/app/model/dotcomrendering/pageElements/PageElement.scala

```diff
@@ -1309,8 +1309,14 @@ object PageElement {
             fields.role,
           )
         }).toList
-      case Code =>
-        List(CodeBlockElement(None, true)) // Force isMandatory to avoid rendering any articles with Codeblocks in AMP
+      case Code => {
+        (for {
+          data <- element.codeTypeData
+        } yield {
+          CodeBlockElement(data.html, true)
```

Comment:
I seem to recall there's a language property or something that DCR is expecting as well